src/composite/MegaChart.js

```diff
@@ -103,7 +106,24 @@
             chartTypeSelect.change = function(a){
                 context.chartType(a.value()).render();
             };
-            this.toolbarWidgets([chartTypeSelect]);
+            
+            chartTypeSelect.change = function(a){
+                context.chartType(a.value()).render();
+            };
+            
+            var csvButton = new Button().value("CSV");
+            csvButton.click = function(a){
+                require(["src/common/Utility"],function(Util){
```

Comment:
Move to main "define" section at the top of the file - in a production environment you will not have saved yourself anything by doing this.